Extensions/Terraform/Src/Tasks/TerraformTaskV2/README.md

```diff
@@ -0,0 +1,82 @@
+# Terraform : Provision infrastructure on Azure, Amazon Web Services(AWS) and Google Cloud Platform(GCP) using the terraform command-line
+
+
+## Overview
+
+This task enables running Terraform commands as part of Azure Build and Release Pipelines. It supports the following Terraform commands
+- init
+- validate
+- plan
+- apply
+- destroy
+
+The following providers are currently supported:
+- AzureRM
+- Amazon Web Services
+- Google Cloud Platform
+
+
+## Contact Information 
+ 
+Please report a problem at [Developer Community Forum](https://developercommunity.visualstudio.com/spaces/21/index.html) if you are facing problems in making this task work.  You can also share feedback about the task like, what more functionality should be added to the task, what other tasks you would like to have, at the same place.
+
+
+## Pre-requisites for the task
+
+
+The only pre-requisite for the task is that Terraform must be installed on the Azure Pipelines build agent. If you want an exact version of Terraform on the agent then you can use the [Terraform tool tnstaller task](https://aka.ms/AA5j5pi)
```

Comment:
Could perhaps update this text to include that only `terraform>=v0.12` will be supported by the backend configuration.

Reply:
Thanks, resolved!